test/symbol-test.js

```diff
@@ -80,8 +80,8 @@ tape("symbol.type(symbolOctagon) generates a polygon with the specified size", f
 
 tape("symbol.type(symbolOctagon) generates the expected path", function(test) {
   var s = shape.symbol().type(extra.symbolOctagon).size(function(d) { return d; });
-  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
-  test.pathEqual(s(10), "M1.880302,0L1.880302,0L1.329574,1.329574L0,1.880302L-1.329574,1.329574L-1.880302,0L-1.329574,-1.329574L0,-1.880302L1.329574,-1.329574Z");
+  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
+  test.pathEqual(s(10), "M1.880302,0L1.329574,1.329574L0,1.880302L-1.329574,1.329574L-1.880302,0L-1.329574,-1.329574L0,-1.880302L1.329574,-1.329574Z");
```

Comment:
<img width="441" alt="1__vim" src="https://cloud.githubusercontent.com/assets/529247/25559121/b4134ba8-2d3d-11e7-97f1-f68924ee5d71.png">
And again.